app/mailers/confirmation_mailer.rb

```diff
@@ -0,0 +1,40 @@
+class ConfirmationMailer < MandrillMailer::TemplateMailer
+  include Rails.application.routes.url_helpers
+  include Devise::Controllers::UrlHelpers
+  default from: 'support@freshfoodconnect.org'
```

Comment:
Do we want to move this into some type of configuration and not have it hard-coded in the source?